src/app/client/src/styles.css

```diff
@@ -1145,7 +1145,12 @@ i.large.icon.limitedPublishingLinkIcon {
   width: auto;
   height: auto;
 }
-
+.tenant-header-logo-before-login {
```

Comment:
is the Css approved by the UI team and the PM

Reply:
Css is updated as per the ui style guide, (Please check the attachment here - https://project-sunbird.atlassian.net/browse/SB-6387)